test/testenv/remote_index_utils.go

```diff
@@ -98,6 +99,18 @@ func GenerateIndexVolumeSpec(volumeName string, endpoint string, secretRef strin
 	}
 }
 
+// GenerateIndexVolumeSpecAzure return VolumeSpec struct with given values for Azure
+func GenerateIndexVolumeSpecAzureManagedId(volumeName string, endpoint string, provider string, storageType string, region string) enterpriseApi.VolumeSpec {
+	return enterpriseApi.VolumeSpec{
+		Name:     volumeName,
+		Endpoint: endpoint,
+		Path:     azureIndexesContainer,
+		Provider: provider,
+		Type:     storageType,
+		Region:   region,
```

Comment:
Region field is not used in Azure blob. it is not required from YAML also. 
if it is set, it is ignored with operator code for azure blob.